Client/Frontend/Home/HistoryHighlights/HistoryHighlightsCell.swift

```diff
@@ -0,0 +1,167 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+private struct RecentlyVisitedCellUX {
+    static let generalCornerRadius: CGFloat = 10
+    static let heroImageDimension: CGFloat = 24
+}
+
+struct RecentlyVisitedCellOptions {
+    let title: String
+    let heroImage: UIImage?
+    let corners: UIRectCorner?
+    let hideBottomLine: Bool
+    let isFillerCell: Bool
+
+    init(title: String,
+         shouldHideBottomLine: Bool,
+         with corners: UIRectCorner? = nil,
+         and heroImage: UIImage? = nil,
+         andIsFillerCell: Bool) {
+
+        self.title = title
+        self.hideBottomLine = shouldHideBottomLine
+        self.corners = corners
+        self.heroImage = heroImage
+        self.isFillerCell = andIsFillerCell
+    }
+
+    init(shouldHideBottomLine: Bool,
+         with corners: UIRectCorner? = nil,
+         andIsFillerCell: Bool) {
+
+        self.init(title: "",
+                  shouldHideBottomLine: shouldHideBottomLine,
+                  with: corners,
+                  and: nil,
+                  andIsFillerCell: andIsFillerCell)
+    }
+}
+
+/// A cell used in FxHomeScreen's History Highlights section.
+class HistoryHighlightsCell: UICollectionViewCell, ReusableCell {
```

Comment:
Thanks for splitting that up in a new file! Much easier to read.

Reply:
For realz!